src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,23 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
```

Comment:
I guess it is better to destruct post. 
export const PostInfo = ({ post }) => {
  const { title, user, body, comments } = post;

Reply:
➕ , would be great to make a destructuring